assets/js/components/KeyMetrics/utils.js

```diff
@@ -58,3 +61,41 @@ export function whenKeyMetricsWidgetVisible() {
 		return WrapperComponent;
 	};
 }
+
+/**
+ * Provides key metric widgets registration data to the given registry.
+ *
+ * This is a temporary utility until a solidified solution is implemented to effortlessly provide
+ * widget registrations to the registry.
+ *
+ * @since n.e.x.t
+ * @see {@link https://github.com/google/site-kit-wp/issues/7264} Initiative to implement a utility to provide widget registrations.
+ *
+ * @param {Object} registry The registry to set up.
+ * @param {Object} widgets  Object containing options mapped to widget slugs.
+ */
+export const provideKeyMetricsWidgetRegistrations = ( registry, widgets ) => {
```

Comment:
As a test-only utility, please move this to a module specific to tests so that we don't bundle it in production. Maybe webpack is smart enough to remove it as dead code but better to keep test sources separate regardless 👍

Reply:
Moved, let me know if it looks good, thank you!